core/src/plugins/dird/python/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ if(Python2_FOUND)
   )
   install(TARGETS python-dir DESTINATION ${plugindir})
   target_link_libraries(python-dir ${Python2_LIBRARIES} bareos)
-  target_include_directories(python-dir PUBLIC SYSTEM ${Python2_INCLUDE_DIRS})
+  target_include_directories(python-dir SYSTEM PUBLIC ${Python2_INCLUDE_DIRS})
```

Comment:
while this will silence the warning for strict-aliasing, we should probably also set `-fno-strict-aliasing` as a compile option so the compiler will not generate broken code.